back/boxtribute_server/graph_ql/definitions/protected/types.graphql

```diff
@@ -93,6 +94,25 @@ type Product {
   deletedOn: Datetime
 }
 
+"""
+Representation of a standard product, containing information about [`ProductCategory`]({{Types.ProductCategory}}), size, and [`ProductGender`]({{Types.ProductGender}}).
+Users can enable this standard product for their bases which creates a standard product instantiation.
+"""
+type StandardProduct {
+  id: ID!
```

Comment:
add value for enabled or not.

Reply:
@HaGuesto since a user can have access to multiple bases, and hence a standard product can be enabled for multiple bases, I decided to return a list of bases here